src/Symfony/Component/String/AbstractUnicodeString.php

```diff
@@ -396,6 +396,14 @@ public function trimStart(string $chars = " \t\n\r\0\x0B\x0C\u{A0}\u{FEFF}"): pa
         return $str;
     }
 
+    public function underscore(): parent
+    {
+        $str = $this->camel()->snake();
+        $str->string = mb_strtolower(preg_replace('~(?<=[a-z])([0-9])~u', '_$1', $str->string), 'UTF-8');
```

Comment:
This regexp is wrong in AbstractUnicodeString, as it only adds an underscore between an ASCII letter and a number, but does not account for any other unicode letter before the number.

Reply:
I believe it's fixed now